### PR TITLE
Feature/single news page

### PR DESCRIPTION
**Pull request by bradybridges**

### CODE REVIEW

File: src/components/socialMediaBar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+          <li id="fb-logo" className="social-media-bar__image"></li>
+        </a>
+        <a href="https://www.twitter.com">
+          <li id="twitter-logo" className="social-media-bar__image"></li>
+        </a>
+        <a href="https://www.gmail.com">
+          <li id="email-logo" className="social-media-bar__image"></li>
+        </a>
+      </ul>
+    </div>
```
Comment (LayaTaal) [inline src/components/socialMediaBar.js]: I'm not really sure how useful this whole thing is. Ideally we want to change this into buttons allowing someone to share the current news story.

Take a look at how Facebook formats share links: https://developers.facebook.com/docs/plugins/share-button/

We can mimic this using the current page url, and follow something similar for Twitter. Email should really be using "mailto: email@address?subject=<page-name>" or something along these lines.

I know this is how it is done on production, but I think this is a worthwhile and small improvement we can go ahead and make.

You can also search for some existing Gatsby plugin to handle social media sharing, and leverage that.

File: src/templates/singleNews.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  const NewsImage = ({ file, description }) => {
+    return (
+      <React.Fragment>
+        {(file || description) && (
+          <div className="news-article__image">
+            {file && (
+              <img
+                src={file["en-US"].url}
+                alt={description ? description["en-US"] : ""}
```
Comment (LayaTaal) [inline src/templates/singleNews.js:66]: Good job including alt text with this!

**Review by LayaTaal — APPROVED** [review]
Looks good @bradybridges! I think we can just improve the social media share a little bit and that is all. Let me know if you want to go over that at all, otherwise good to go!

File: src/components/socialMediaBar.js (hunk 1 above)
Comment (bradybridges) [reply src/components/socialMediaBar.js]: I agree that would be a nice improvement, the current social media bar isn't very helpful at the moment. I'll look into updating that to share the current news article on FB/Twitter/email.

**Pull request merged**